### PR TITLE
Prepare the ruby library for v1.5.0

### PR DESCRIPTION
**Pull request by jakedipity**
This also cleans up the CHANGELOG

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,37 @@
...
+
+## [v1.4.0] - 2020-01-30
+### Added
+- Everything.
+
+[Unreleased]: https://github.com/logdna/ruby/compare/v1.5.0...master
+[v1.5.0]: https://github.com/logdna/ruby/compare/v1.4.2...v1.5.0
+[v1.4.2]: https://github.com/logdna/ruby/compare/v1.4.1...v1.4.2
+[v1.4.1]: https://github.com/logdna/ruby/compare/v1.4.0...v1.4.1
+[v1.4.1]: https://github.com/logdna/ruby/releases/tag/v1.4.0
```
Comment (smusali) [inline CHANGELOG.md]: @jakedipity, small fix: `[v1.4.1]` should be replaced by `[v1.4.0]`

**Review by smusali — CHANGES_REQUESTED** [review]
just a small request

File: CHANGELOG.md (hunk 1 above)
Comment (smusali) [reply CHANGELOG.md]: Ooooops...

**Review by smusali — APPROVED** [review]
lgtm!

File: CHANGELOG.md (hunk 1 above)
Comment (jakedipity) [reply CHANGELOG.md]: Sorry, I'm rapid firing small fixes haha. Should be fixed now!

**Review by jmoses — APPROVED** [review]

**Pull request merged**